cobbler/utils.py

```diff
@@ -567,32 +567,32 @@ def input_string_or_list(options: Optional[Union[str, list]]) -> Union[list, str
     return input_string_or_list_no_inherit(options)
 
 
-def input_string_or_dict_no_inherit(
-    options: Union[str, list, dict], allow_multiples=True
-) -> Union[str, Tuple[bool, dict]]:
-    """
-    See :meth:`~cobbler.utils.input_string_or_dict`
-    """
-    if options == enums.VALUE_INHERITED:
-        return enums.VALUE_INHERITED
-    return input_string_or_dict(options, allow_multiples)
-
-
 def input_string_or_dict(
     options: Union[str, list, dict], allow_multiples=True
-) -> Tuple[bool, dict]:
+) -> Union[str, dict]:
```

Comment:
We talked about this before, this is another instance where https://github.com/cobbler/cobbler/issues/3074 would improve the clarity of the code.

Reply:
Yes indeed but sadly we don't have it yet. Thus I wanted to limit myself to the bugfix at the moment.